### PR TITLE
the address-allocation feature flag was removed...

### PR DESCRIPTION
**Pull request by matthewhelmke**
...so the references to it were also removed. That's the only change here.

@juju/docs for confirmation before merge

### CODE REVIEW

File: src/en/reference-release-notes.md [hunk 1]
```diff
@@ -1713,25 +1704,17 @@ The versions covered here are:
...
-  MAAS providers also supports networking on KVM. Containers and Virtual
-  Machines will be given statically allocated private IP addresses from
-  the same subnet as their host machine. For example on MAAS you can now:
-
-  ```bash
-  juju deploy wordpress --to lxc:0
-  juju add-unit mysql --to kvm:1
-  ```
-
-  This experimental feature can be enabled when the environment is
```
Comment (matthewhelmke) [inline src/en/reference-release-notes.md]: I only intended to delete this part, not the above.

File: src/en/reference-release-notes.md [hunk 2]
```diff
@@ -1713,25 +1704,17 @@ The versions covered here are:
...
-  bootstrapped like so:
-
-  ```no-highlight
-  JUJU_DEV_FEATURE_FLAGS="address-allocation" juju bootstrap
-  ```
+ ### Experimental: Addressable LXC Containers and KVM Instances on AWS and MAAS
+ 
+ The Juju AWS and MAAS providers now support starting LXC containers. The
+ MAAS providers also supports networking on KVM. Containers and Virtual
```
Comment (pmatulis) [inline src/en/reference-release-notes.md]: provider (singular)

Comment (dimitern) [inline src/en/reference-release-notes.md]: This whole section should be gone as well in 2.0 rel notes

Comment (matthewhelmke) [inline src/en/reference-release-notes.md]: Thanks, @dimitern , I'll take care of it.

**Pull request merged**